proposal.md

```diff
@@ -2487,6 +2655,38 @@ template <> struct hash<json_v1::json>
 *Effect:* The template specialization to calculate the hash value for the specified
 JSON value.
 
+<a name="limitations"></a>
+## Limitations and Interoperability

```

Comment:
This chapter was missing indeed.

The question remains: what are the limitations set by the proposal.

Regarding interoperability, I think it's the right choice to support UTF-8 but not UTF-16 and UTF-32 for the sake of simplicity.